oauth2client/crypt.py

```diff
@@ -248,3 +237,25 @@ def verify_signed_jwt_with_certs(jwt, certs, audience=None):
     _check_audience(payload_dict, audience)
 
     return payload_dict
+
+
+def configure_module():
+    """Configure module-wide variables.
+
+    Not all testing frameworks load modules directly, so we need a way to
+    define these on set up.
+    """
+    global Signer
+    global Verifier
```

Comment:
Guido's hack only works on Python 3, so that is not an option.

I was incorrect in my statement that Pytest didn't load module variables, modules are just reused between tests.  I've added some setUp/tearDown code to do some housecleaning when module variables are changed (at least for `oauth2client.crypt`)

@dhermes Thanks for making me take another look at things!